dist/READMEopenshifttechpreview.md

```diff
@@ -138,14 +138,133 @@ or
 $ make fedora
 ```
 
-In a development cycle, a new image can be built and pushed and the ovnkube-master and ovnkube daemonsets
-can be deleted and recreated.
+## Development Cycle:
+In a development cycle, a new image can be built and pushed and the ansible
+scripts can uninstall and reinstall ovn on the cluster.
 
 ```
-# cd ovn-kubernetes/dist/yaml
+# cd ansible
+# ./run-playbook uninstall
+# ./run-playbook
+```
+
+Alternatively once the daemonsets are running, the pods can be deleted. They
+will be automatically created using the new image.
+```
 # oc project ovn-kubernetes
-# oc delete -f ovnkube.yaml
-# oc delete -f ovnkube-master.yaml
-# oc create -f ovnkube-master.yaml
-# oc create -f ovnkube.yaml
+# oc get po
+NAME                   READY     STATUS    RESTARTS   AGE
+ovnkube-6sk8k          3/3       Running   0          1m
+ovnkube-9gc8j          3/3       Running   0          1m
+ovnkube-ks8kt          3/3       Running   0          1m
+ovnkube-master-4nnkh   4/4       Running   0          1m
+# oc delete po ovnkube-6sk8k ovnkube-9gc8j ovnkube-ks8kt ovnkube-master-4nnkh
 ```
+
+##Debugging Aids
+
+The ovnkube pod has the following containers: ovs-daemons ovn-controller
+ovn-node The ovnkube-master pod has the following containers: run-ovn-northd
+nb-ovsdb sb-ovsdb ovnkube-master
+
+Logs from the containers can be viewed using the "oc logs" command. Each
```

Comment:
Or, technically, 'kubectl logs' works too right?

Reply:
'kubectl logs' works, I changed the text.